### DISCUSSION TITLE
Error creating certificates

### DISCUSSION THREAD

**Discussion by jasenf** [rcl-ssl/docs#44] [Q&A] [answered]
Hi,

We've been using LetsEncrypt for years.  I logged in today to renew a few certificates and am getting errors on all cases.  I'm just trying to create a new cert now:

Alone SAN certificate : xxxx,*.xxxx with challenge : DNS Certificate in chain not found, issuer :O=Digital Signature Trust Co.,CN=DST Root CA X3, subject : C=US,O=Internet Security Research Group,CN=ISRG Root X1

Is there an outage at the moment or has something changed?

**Comment by jasenf** [comment]
Update:  I can not create a single certificate any more, no matter what domain, I get the same error as above.  I've been doing this same process for years, something seems broken within LetsEncrypt.  Please help! I have certs that expire in a few days.

**Comment by rcladmin** [comment]
I am looking into this

**Comment by rcladmin** [comment, marked as answer]
Try it now, It should be working.

**Comment by jasenf** [comment]
Thank you for fixing this!  very much appreciated.
